WalletWasabi.Fluent/Extensions/TaskExtension.cs

```diff
@@ -0,0 +1,24 @@
+using System.Threading.Tasks;
+using WalletWasabi.Logging;
+
+namespace WalletWasabi.Fluent.Extensions;
+
+public static class TaskExtension
+{
+	public static void FireAndForget(this Task task)
+	{
+		_ = RunAsync(task);
+	}
```

Comment:
Very Ultra Nit: In this case thank God, we are not forgeting the task. Should this be `FireAndRemember` or something similarly creative :D?

Reply:
Ah, in my mind I tought because we await it in `RunAsync` this logic could be appliable, but okay, dismiss then.